openlane_run/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Run design with latest OpenLane release
+
+inputs:
+  design_name:
+    description: Name of design to run
+    required: true
+  pdk_root:
```

Comment:
couldn't this just assume PDK_ROOT? assuming cache_pdks is always present as a siblings action in the workflow?

Reply:
Yes, but I think to require it as input is cleaner. We won't have to rely on the underlying tool to provide a meaningful error message or the user to know how to fix it.

Reply:
maybe it should default on `PDK_ROOT` then, or get it from `cache_pdks` inputs? (so that it's less burden on the users to have them always match)